llvm/test/CodeGen/AMDGPU/amdgcn.bitcast.inreg.224bit.ll

```diff
@@ -0,0 +1,2262 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+
+; RUN: llc -mtriple=amdgcn < %s | FileCheck -check-prefix=GCN %s
```

Comment:
Avoid using the default target, use an explicit name for it. GCN is usually the shared prefix, but you're not using that. Also replace the check-prefixes with check-prefix if you're only using one